samples/dnn/inpainting.cpp

```diff
@@ -0,0 +1,219 @@
+/*
+This sample inpaints the masked area in the given image.
+
+Copyright (C) 2025, Bigvision LLC.
+
+How to use:
+    Sample command to run:
+
+        ./example_dnn_inpainting
```

Comment:
the comment should describe how to provide custom input, e.g.

`./example_dnn_inpainting [--input=<image_name>]`